### PR TITLE
ServiceAccount Member Dashboard Access

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
allow all kind of members to access the dashboard (including ServiceAccount members)
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement user

```

### CODE REVIEW

**Comment by holgerkoser** [comment]
We should add a unit test `services.projects.spec.js` for projects make sure the list of projects is always correct.

**Comment by grolu** [comment]
> We should add a unit test `services.projects.spec.js` for projects to make sure the list of projects is always correct.

done

File: backend/test/services.projects.spec.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+//
+
+'use strict'
+
+const delay = require('delay')
+const projects = require('../lib/services/projects')
+const cache = require('../lib/cache')
+const authorization = require('../lib/services/authorization')
+
+const { expect } = require('chai')
```
Comment (holgerkoser) [inline backend/test/services.projects.spec.js:14]:
```suggestion
```

File: backend/test/services.projects.spec.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
+//
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+'use strict'
+
+const delay = require('delay')
```
Comment (holgerkoser) [inline backend/test/services.projects.spec.js:9]:
```suggestion
```

File: backend/test/services.projects.spec.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    const createUser = (username) => {
+      return {
+        id: username
+      }
+    }
+
+    beforeEach(function () {
+      sandbox.stub(cache, 'getProjects').returns(projectList)
+      sandbox.stub(authorization, 'isAdmin').callsFake(async (user) => {
+        await delay(1)
```
Comment (holgerkoser) [inline backend/test/services.projects.spec.js:92]:
```suggestion
        await  nextTick()
```

**Review by holgerkoser — APPROVED** [review]
proposed some small cosmetic changes but it
/lgtm

**Review by petersutter — APPROVED** [review]
/lgtm

**Pull request merged**